packages/badge/src/lib/Badge.stories.tsx

```diff
@@ -1,18 +1,17 @@
+import React from 'react';
 import type { StoryObj } from '@storybook/react';
 import { Badge, BadgeProps } from './Badge';
 
 export default {
   title: 'Components/Badge/Badge',
   component: Badge,
-  tags: ['autodocs']
+  tags: ['autodocs'],
 };
 
 export const _Badge: StoryObj<typeof Badge> = {
-  render: (args: BadgeProps) => (
-    <Badge {...args} />
-  ),
+  render: (args: BadgeProps) => <Badge {...args} />,
   args: {
     children: 'This text is a child of the badge',
-    badgeContent: 'badge content'
-  }
+    badgeContent: 3,
+  },
```

Comment:
I'd probably put variant in here as well so it's higher on the controls list since the dot variant is a common use it seems

Reply:
done